flycheck-haskell.el

```diff
@@ -80,7 +80,16 @@
       (stack-exe
        `(,stack-exe "--verbosity" "silent" "runghc" "--no-ghc-package-path" "--" "--ghc-arg=-i"))
       (runghc-exe
-       `(,runghc-exe "-i"))
+       ;; Dependencies needed by get-cabal-configuration.hs
+       `(,runghc-exe "--"
+                     "-i"
+                     "-packageCabal"
+                     "-packagebase"
+                     "-packagebytestring"
+                     "-packagecontainers"
+                     "-packageprocess"
+                     "-packagedirectory"
+                     "-packagefilepath"))
```

Comment:
Why it became required to list these packages explicitly?

Reply:
:man_shrugging: I needed to specify those dependencies in order to run the script. I'm not sure either how the dependencies were found without nix-style builds. I guess they would be made available from the global dependency database.

I noticed that if I run flycheck on a project where those dependencies are already specified then there is no need to list them there.